components/button/button.css

```diff
@@ -0,0 +1,59 @@
+.btn{
+    padding: 0.5rem 0.75rem;
+    font-size: 1rem;
+    border-radius: 0.25rem;
+    border:none;
+}
+.btn-link{
+    padding: 0.5rem 0.75rem;
+    font-size: 1rem;
+    text-decoration: none;
+    border-radius: 0.25rem;
+}
+.btn-primary{
+    color: var(--background-color);
+    background-color: var(--primary-color);
+    border:0.125rem solid var(--primary-color);
+}
+.btn-secondary{
+    color: var(--secondary-color);
+    background-color: var(--background-color);
+    border:0.125em solid var(--secondary-color);
+}
+.btn-primary:hover:enabled{
+    box-shadow:  4px 4px 6px 0 rgba(255,255,255,.5),
+              -4px -4px 6px 0 rgba(216, 227, 240, 0.5), 
+    inset -4px -4px 6px 0 rgba(255,255,255,.2),
+    inset 4px 4px 6px 0 rgba(231, 243, 247, 0.4);
+    border-radius: 4px;
+}
+.btn-secondary:hover:enabled{
+    color: #fff;
```

Comment:
Use css variables

Reply:
Thanks for your comment. It is fixed now